topics/admin/tutorials/connect-to-compute-cluster/slides.html

```diff
@@ -296,128 +296,7 @@
 
 ---
 
-# Galaxy Dependency Resolver Configuration
```

Comment:
@hexylena The text removed here contained more (I think useful) material than what's in `topics/admin/tutorials/toolshed/slides.html` . Also I've been thinking what's the more appropriate slide deck to include such info about dependency resolution, and I feel it should be in `topics/admin/tutorials/tool-management/slides.html` (there was one slide, which I removed in d5ba4aa1f5d8c6e2ab3ef4e63897478702a958c6 :blush:).

What do you think? I can do the work if you agree.